packages/core/src/tracing/constants.js

```diff
@@ -23,15 +23,20 @@ exports.kafkaLegacyTraceLevelHeaderName = 'X_INSTANA_L';
 exports.kafkaLegacyTraceLevelValueSuppressed = Buffer.from([0]);
 exports.kafkaLegacyTraceLevelValueInherit = Buffer.from([1]);
 
-// new kafka trace correlation (string values) starting approximately 2021-11
+// new kafka trace correlation (string values) starting approximately 2022-10
```

Comment:
It says 2023 below

> (starting approximately October 2023)

Reply:
October 2022 refers to phase 1, where we start to inject both sets of headers by default. October 2023 refers to phase 2, where switch over to only send string headers.

Reply:
It is a bit ambigious. I'll make that more precise.

Reply:
Fixed, the new wording should be a bit clearer.